kernel/fatal.c

```diff
@@ -117,5 +119,30 @@ void z_fatal_error(unsigned int reason, const z_arch_esf_t *esf)
 		__ASSERT(!k_is_in_isr(),
 			 "Attempted to recover from a fatal error in ISR");
 	}
+
+#if defined(z_arch_is_in_nested_exception)
+	/* - If k_sys_fatal_error_handler() has returned, and
+	 * - ARCH supports checking whether the fault has occurred
+	 *   in interrupt context,
+	 * we perform the check below, to allow us to return without
+	 * aborting the current thread.
+	 *
+	 * There is one exception to this policy: a stack sentinel
+	 * check may be performed (on behalf of the current thread)
+	 * during ISR exit, but in this case the thread should be
+	 * aborted.
+	 */
+	if (z_arch_is_in_nested_exception(esf)) {
+		LOG_ERR("Fault during interrupt handling\n");
```

Comment:
This printout needs to be moved up, before k_sys_fatal_error_handler() is called

Reply:
ok ;)

Reply:
Done